crates/bench/benches/special.rs

```diff
@@ -62,20 +63,23 @@ fn custom_db_benchmarks(c: &mut Criterion) {
     let module = runtime.block_on(async { BENCHMARKS_MODULE.load_module(config, None).await });
     let mut group = c.benchmark_group("special");
     // This bench take long, so adjust for it
-    group.measurement_time(Duration::from_secs(60 * 2));
     group.sample_size(10);
```

Comment:
Is this necessary or can be removed as well in favour of defaults? Also the comment above should be removed I think.

Reply:
Without limiting the sample the test goes from ~2min to ~10min in my machine, without any major improvement in the result.

Reply:
Hm for me it doesn't, Criterion still defaults to same 10 samples...

Reply:
But fair enough.

Reply:
Ah maybe it's because you changed params in the circle benchmark to larger ones but I didn't locally.